install.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+set -u
+
+#The SBO can also be installed in a vanilla kubernetes cluster. A prerequisite for this would be to add credentials for the registry.redhat.io in the cluster. Steps to be followed to achieve the same are listed below:
+#1) Follow [Red Hat Container Registry Authentication Steps](https://access.redhat.com/RegistryAuthentication)
+#2) Verify that your credentials are correct using docker login -u <your_username> -p <your_passwd> registry.redhat.io
+docker login registry.redhat.io
+
+EXIT_CODE=$?
+if [ $EXIT_CODE -ne 0 ];
+then
+  echo "docker login was not successful. Please try again"
+  exit 0
+fi
```

Comment:
in case of unsuccessful login, we are not going to reach this block, because shebang line is `#!/bin/bash -e` - `-e` arg makes that scripts exits on any error immediately.

Reply:
In that case we can omit checking the exit code? https://github.com/redhat-developer/service-binding-operator/pull/738#discussion_r517880610

Reply:
right, lines 9-14 will never be executed.

Reply:
Done